packages/piavpn/piavpn.pacscript

```diff
@@ -0,0 +1,22 @@
+name="piavpn"
+breaks="${name}-git"
+repology=("project: ${name}")
+version="3.3-06906"
+url="https://installers.privateinternetaccess.com/download/pia-linux-${version}.run"
+description="Private Internet Access VPN Client"
+hash="75efbf335af99673f55c81ff5f7e70d66ced52913ace2baca12253e19d3e64cf"
+maintainer="WRM-42 <y8bsbahy@anonaddy.me>"
+
+prepare() {
+    true
+}
+
+build() {
+    true
+}
+
+install() {
+    chmod +x pia-linux-${version}.run
```

Comment:
Nothing gets installed to `/usr/src/pacstall`, which results in a stow error, which breaks the build

Reply:
Yea I think this should actually follow the PKGBUILD of piavpn-bin properly so it works correctly with pacstall since the AUR package extracts the files from the run file and then installs them manually by copying the files to the locations.